docs/guide/personalization/recommendation_client.md

```diff
@@ -14,46 +14,75 @@ the Personalization service, see [Developer guide](developer_guide/tracking_api.
 
 Before you can use the Personalization service, you must configure the client.
 
-### Set up Content Type tracking
+### Set up item type tracking
 
 For the recommendations to be calculated, apart from visitor events (CLICK, BUY, etc.), 
-the Recommendation engine must be fed with the list of Content Types that are tracked.
+the Personalization server must be fed with the list of item types that are tracked.
 
-You define Content Types to be tracked in the `config/packages/ezplatform.yaml` file.
+You define item types to be tracked in the `config/packages/ezplatform.yaml` file.
 The content is then initially exported by a script.
 After this, it is synchronized with the Personalization service every time a change 
 occurs in the Back Office.
 
-The client's configuration is SiteAccess-aware, and can resemble the following example:
+The recommendation client's configuration is SiteAccess-aware.
+If your installation [hosts multiple sites](https://doc.ibexa.co/projects/userguide/en/latest/personalization/use_cases/#multiple-stores) with different customer IDs, 
+for example, to provide separate recommendations for different language versions 
+of the site, provide the credentials that correspond to each of the sites.
+
+The configuration can resemble the following example:
 
 ``` yaml
 ezrecommendation:
     system:
-        <siteaccess>:
+        <site_access_name_1>:
+            site_name: '<site_name_1>' # For example 'ENU store'
+            host_uri: `'%env(RECOMMENDATION_HOST_URI)%'`
             authentication:
-                customer_id: 12345
-                license_key: 1234-5678-9012-3456-7890
-            included_content_types: [blog, article]
-            random_content_types: [blog]
-            host_uri: http://example.com
+                customer_id: `'%env(RECOMMENDATION_CUSTOMER_ID)%'`
+                license_key: `'%env(RECOMMENDATION_LICENSE_KEY)%'`
+            export:
+                authentication:
+                    method: 'basic'
+                    login: `'%env(RECOMMENDATION_CUSTOMER_ID)%'`
+                    password: `'%env(RECOMMENDATION_LICENSE_KEY)%'`
+            included_item_types: [product, article]
+            random_item_types: [blog]
+
+        <site_access_name_1>:
+            site_name: '<site_name_2>' # For example 'FRA store'
+            host_uri: `'%env(FRA_HOST_URI)%'`
+            authentication:
+                customer_id: `'%env(FRA_CUSTOMER_ID)%'`
+                license_key: `'%env(FRA_LICENSE_KEY)%'`
+            export:
+                authentication:
+                    method: 'basic'
+                    login: `'%env(FRA_CUSTOMER_ID)%'`
+                    password: `'%env(FRA_LICENSE_KEY)%'`
```

Comment:
Method **user** requires login and password parameters. I think better idea is suggest to use custom login and self-generated password. Both options can be stored in ENV's as well. 
```suggestion
                    method: 'user'
                    login: '%env(FRA_CUSTOM_EXPORT_LOGIN)%'
                    password: '%env(FRA_CUSTOM_EXPORT_PASSWORD)%'
```